docs/source/Plugin/P012.rst

```diff
@@ -32,19 +82,26 @@ Supported hardware
 
 |P012_usedby|
 
-.. Commands available
-.. ^^^^^^^^^^^^^^^^^^
+Example photo of a 4x20 (LCD2004) display:
 
-.. .. include:: P012_commands.repl
+.. image:: P012_LCD_4x20.jpg
 
-.. Events
-.. ~~~~~~
+**Attention** When using a 5V powered display with on-board pull-up resistors on the I2C bus, a level-convert should be used to connect to an ESP to avoid damage to the 3.3V I2C bus on the ESP!

```

Comment:
Not only the ESP, but also any other device connected to the I2C bus.

Reply:
I'll reword that a bit to be more clear, and probably use the yellow colored button attribute to make 'Attention' jump out.

Reply:
I think RTD has a "note" or "warning" style.